app/assets/stylesheets/modules/error.scss

```diff
@@ -0,0 +1,90 @@
+.error {
+  color: $error-color;
+  height: 400px;
+  text-align: left;
+  padding-top: 210px !important;
+  position: relative;
+  a { color: $error-color; }
+
+  @include mq-tablet() {
+    padding-top: 155px !important;
+  }
+
+  @include mq(880px) {
+    padding-top: 120px !important;
+    background: none;
+  }
+
+  @include mq(320px) {
+    padding-top: 16px !important;
+    height: 550px;
+  }
+}
+
+.error-img {
+  position: absolute;
+
+  @include mq(880px) {
+    display: block;
+    position: absolute;
+    left: 50%;
+    margin-left: -125px;
+  }
+}
+
+.error-code {
+  font-weight: bold;
+  font-size: 90px;
+  margin: 110px 48px;
+  position: absolute;
+
+  @include mq(880px) {
+    position: absolute;
+    float: none;
+    left: 50%;
+    margin-left: -77px;
+  }
+
+}
+
+.error-title {
+  font-weight: 300;
```

Comment:
be consistent, on other places you're using the string value (e.g. bold) instead the integer value (e.g. 300); make sure you use the same ...